sbnci/scripts/sbnciplots.sh

```diff
@@ -30,4 +31,4 @@ ln -v ci_validation_histos.root ci_validation_histos_${SBNDCODE_VERSION}.root
 
 ls -lh
 
-root -l -b -q $comparisonScript\(\"${SBNDCODE_VERSION}\",\"${ref_sbndcode_version}\"\)
+root -l -b -q $comparisonScript\(\"${SBNDCODE_VERSION}\",\"${ref_sbndcode_version}\",${normalise_plots}\)
```

Comment:
Here I'd replace this call with
`root -l -b -q $comparisonScript\(\"${SBNDCODE_VERSION}\",\"${ref_sbndcode_version}\",${normalise_plots:-false}\)`

in order to have a default value, in this case I choose `false` as default. In case a default value is missing, there is a trailing comma that could cause issue if `normalise_plots` is not set.